src/Jira/Api.php

```diff
@@ -469,6 +470,10 @@ public function api(
 
         if (strlen($result)) {
             $json = json_decode($result, true);
+            if (!is_array($json)) {
```

Comment:
Better to check for `null` specifically as this clearly indicates the error. Not sure if that won't backfire if some of API calls really has returns null.